src/platformdirs/api.py

```diff
@@ -58,8 +58,9 @@ def __init__(  # noqa: PLR0913
         """
         self.multipath = multipath
         """
-        An optional parameter only applicable to Unix/Linux which indicates that the entire list of data dirs should be
-        returned. By default, the first item would only be returned.
+        An optional parameter which indicates that the entire list of data dirs should be returned.
+        By default, the first item would only be returned. This is only applicable for Linux/Unix systems,
+        as well as macOS systems where `Homebrew <https://brew.sh>`_ is installed.
```

Comment:
Remove this, IMHO this addendum belongs to macos.py only.

Reply:
I removed the references to both macOS and Linux/Unix.